contentG.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html> 
+<html lang="en">
+	<head>
+		<title>Classical Antiquity</title>
+<!-- Meta Data -->
+		<meta charset="utf-8">
+		<meta name="Keywords" content="Classical World, Ancient Rome, Ancient Greece"/>
+		<meta name="Author" content="Thanadon Bunsukha"/>
+		<meta name="Description" content="Classical Antiquity Website"/>
+		<meta name="Viewport" content="width=device-width, intitial-scale=1">
+<!-- Webpage's stylesheet -->
+		<link rel="stylesheet" href="css/style.css">
+	</head>
+	<body>
+		<div class ="content">
+			<div class="box_1" style="border-bottom: 3px solid #1f456e;">
+				<nav>
+					<ul>
+						<li><a href="index.html">Home</a></li>
+						<li><a href="contentG.html">Ancient Greece</a></li>
+						<li><a href="contentR.html">Ancient Rome</a></li>
+					</ul>
+				</nav>
+			</div>
+			<div class="box_2">
+			</div>
+			<div class="box_3">
+				<img src="images/Acropolis.jpg" alt="The Acropolis" width="100%" height="100%" style="vertical-align:bottom;">
+			</div>
+			<div class="box_4" style="background-image: linear-gradient(to bottom right, #1f456e, #acace6, #66023c);">
+				<div class="box_4_content" style="border-left: 3px solid #1f456e; border-right: 3px solid #1f456e;">

```

Comment:
This is the final build of the content page. The old code was completely removed and replaced with different ones to allow the website to be more responsive.